public/docs/getting_started/project.md

```diff
@@ -1,41 +1,337 @@
-# Creating a project
+# Quickstart
 
-To use most Truffle commands, you need to run them against an existing Truffle project. This section will show you how to create a new project.
+This page will take you through the basics of creating a Truffle project and deploying a smart contract to a blockchain.
 
-For those just getting started, we have many boilerplates and example applications available to you through [Truffle Boxes](/boxes). 
+## Creating a project
 
-You can start with our MetaCoin example by running the following on the command line:
+To use most Truffle commands, you need to run them against an existing Truffle project. So the first step is to create a Truffle project.
 
-```shell
-mkdir MetaCoin
-cd MetaCoin
-truffle unbox metacoin
-```
+You can create a bare project template, but for those just getting started, you can use [Truffle Boxes](/boxes), which are example applications and project templates. We'll use the [MetaCoin box](/boxes/metacoin), which creates a token that can be transferred between accounts:
```

Comment:
We include the link to details on the MetaCoin box here, but perhaps a brief paragraph to explain what the MetaCoin box does is worthwhile?

Also, I think we unintentionally position boxes as only for those just getting started here, whereas they're beneficial for expert users as well.